cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/CradleEntity.java

```diff
@@ -31,21 +31,29 @@
 	public static final String FIELD_COMPRESSED = "compressed";
 	public static final String FIELD_LABELS = "labels";
 	public static final String FIELD_CONTENT = "z_content";
+	public static final String FIELD_CONTENT_SIZE = "z_content_size";
```

Comment:
z_content_size should describe cassandra real content size (compressed or uncompressed)

instead of z_content_compressed_size track z_content_uncompressed size that shows cradle data original size